.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a bug in tutor-contrib-webhook-receiver
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Summary
+
+Replace this with a short summary (one sentence or short paragraph) of the issue you are seeing.
+
+## What I did
+
+Explain the steps you undertook that led to the unexpected behavior you are observing. Please include relevant documentation snippets, such as the `WEBHOOKRECEIVER__*` settings from Tutor’s `config.yml`.
+
+It’s OK to remove or mask credentials (like your Shopify API key or WooCommerce secret), but otherwise try to err on the side of sharing more rather than less information.
+
+## What I expected to happen
+
+Explain how you expected Tutor to behave with your configuration.
+
+## What actually happened
+
+Explain how Tutor behaved differently than you expected.
+
+## My environment
+
+Please add some information about the environment that you’re working in. At a minimum, include these items:
+
+* Tutor version: (Insert the Tutor version you are using.)
+* `tutor-contrib-webhook-receiver` version: (Insert the plugin version you are using. If you installed from Git, enter the tag or branch name, or the commit reference.)
+
```

Comment:
Should we remind people to include details of their Shopify or WooCommerce product or webhook configuration, if they deem them relevant?